index.ejs

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
```

Comment:
I guess this file is intended to follow `http://www.embeddedjs.com/` thus name `.ejs`?